### PR TITLE
Use original cross_entropy and re-open the loss check in unit test

### PR DESCRIPTION
**Pull request by pkuzyc**
For the loss check in unit test, replace the `tensor_parallel.vocab_parallel_cross_entropy` loss function, use original cross_entropy in Paddle, and re-open the loss check in unit test.

### CODE REVIEW

**Review by From00 — APPROVED** [review]
LGTM
